packages/core/src/payment/payment-strategy-action-creator.spec.ts

```diff
@@ -104,12 +104,11 @@ describe('PaymentStrategyActionCreator', () => {
             ),
             new HostedFormFactory(store),
         );
-        noPaymentDataStrategy = new NoPaymentDataRequiredPaymentStrategy(store, orderActionCreator);
+        noPaymentDataStrategy = new NoPaymentDataRequiredPaymentStrategy(paymentIntegrationService);
```

Comment:
I am thinking can we directly use `createNoPaymentStrategy` here?

Reply:
I have replaced all `new NoPaymentDataRequiredPaymentStrategy()` with `createNoPaymentStrategy()` in `core`.
So only `new NoPaymentDataRequiredPaymentStrategy()` in the `no-payment-integration` now.